client/signup/locale-suggestions/index.jsx

```diff
@@ -1,16 +1,16 @@
 /**
  * External dependencies
  */
-var i18n = require( 'i18n-calypso'),
-	React = require( 'react' ),
-	assign = require( 'lodash/assign' );
+import i18n from 'i18n-calypso';
+import React from 'react';
+import assign from 'lodash/assign';
```

Comment:
We use a [babel plugin](https://github.com/lodash/babel-plugin-lodash) that allows tree-shaking of lodash utils already, so this could be converted to:
```js
import { assign } from 'lodash';
```
FFTI if you feel this is out of this PR's scope though :)